platform/darwin/test/MGLImageSourceTests.m

```diff
@@ -0,0 +1,42 @@
+#import <XCTest/XCTest.h>
+
+#import <Mapbox/Mapbox.h>
+
+@interface MGLImageSourceTests : XCTestCase
+
+@end
+
+@implementation MGLImageSourceTests
+
+
+- (void)testMGLImageSourceWithImageURL {
+
+    MGLCoordinateQuad quad = { { 80, 37}, { 80, 39}, { 81, 39}, { 81, 37}};
```

Comment:
In this test we dont really care about how an image is placed, so the coordinate order can be fixed for someone else to understand.
In general though the corner coordinates are relative to the image, and can be used to rotate an image for placement. The top left corner does not have to be the north eastern coordinate.